eZ/Publish/API/Repository/PermissionCriterionResolver.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository;
+
+/**
+ * This service provides methods for resolving criterion permissions.
+ */
+interface PermissionCriterionResolver
+{
+    /**
+     * Get criteria representation for a permission.
+     *
+     * Will return a criteria if current user has limited access to the given module/function,
+     * however if user has either full or no access then boolean is returned.
+     *
+     * @uses \eZ\Publish\API\Repository\PermissionResolver::hasAccess()
```

Comment:
I'm not sure it's valid to hint that on the interface as this depends on the implementation.

Reply:
true